terraform/gcp/security/gcp-project-member-default-service-account-iam-binding.yaml

```diff
@@ -0,0 +1,25 @@
+rules:
+    - id: gcp-project-member-default-service-account-iam-binding
+      patterns:
+        - pattern: resource
+        - pattern-inside: |
+              resource "google_project_iam_binding" "..." {
+              ...
+              members = [..., $MEMBER, ...]
+              ...
+              }
+        - metavariable-regex:
+            metavariable: $MEMBER
+            regex: ((.*)-compute@developer.gserviceaccount.com)

```

Comment:
Same question as above

Reply:
The default service account structure is `PROJECT_NUMBER-compute@developer.gserviceaccount.com` (https://cloud.google.com/compute/docs/access/service-accounts#default_service_account).